spaceship/lib/spaceship/tunes/language_converter.rb

```diff
@@ -8,6 +8,12 @@ def from_itc_to_standard(from)
           (result || {}).fetch('locale', nil)
         end
 
+        # Converts the Language "UK English" to itc locale en-GB
+        def from_standard_to_itc_locale(from)
```

Comment:
What do you think about using "itc_locale" if it exists but falling back to "locale" if not? Then we'd only need to override "itc_locale" when they differ?